Task1a.trigger

```diff
@@ -0,0 +1,8 @@
+trigger Task1a on Contact (before insert, before update) {
```

Comment:
@lubomyrV 
Why did you choose 'before', but not 'after'? 
see Triggers and Order of Execution https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_triggers_order_of_execution.htm